.github/workflows/release-go-task.yml

```diff
@@ -235,6 +235,7 @@ jobs:
 
   create-release:
     runs-on: ubuntu-latest
+    environment: prod
```

Comment:
```suggestion
    environment: production
```

Reply:
The role can be assumed only by `repo:arduino/arduino-cli:environment:production`